preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -965,8 +1000,47 @@ def run(config: Config) -> None:
                     f"Processing failed. Traceback : {e}. Unprocessed data: {unprocessed}"
                 )
                 continue
+
+            if config.embl_file:
+                for submission_data in processed:
+                    try:
+                        file_content = create_flatfile(
+                            config,
+                            submission_data.processed_entry.accession,
+                            submission_data.processed_entry.version,
+                            submission_data.processed_entry.data.metadata,
+                            submission_data.processed_entry.data.unalignedNucleotideSequences,
+                            submission_data.annotations,
+                        )
+                        file_name = f"{submission_data.processed_entry.accession}.{submission_data.processed_entry.version}.embl"
+                        upload_info = request_upload(submission_data.group_id, 1, config)[0]
+                        file_id = upload_info.fileId
+                        url = upload_info.url
+                        upload_embl_file_to_presigned_url(file_content, url)
+                        submission_data.processed_entry.data.files = {
+                            "annotations": [
+                                FileIdAndName(fileId=file_id, name=file_name)
+                            ]
+                        }
+                    except Exception as e:
+                        submission_data.processed_entry.errors.append(
```

Comment:
wupps these errors will never be sent to the user as that only happens in `submit_processed_sequences` - we will see that in the logs but I think that is not enough. We need to somehow be notified if the files cannot be uploaded.

I think we should retry on a failed upload but if that still doesn't work we need to think about how we handle this case

@fhennig or @corneliusroemer do you have ideas on how to ensure we always know when the file upload fails?

My suggestions are:
1. raise error in prepro (have a failing pod that we should get notified about -> but also bad for other users
2. send a slack notification
3. return the entry in `submit_processed_sequences` with this error so that users will contact us